python/runfiles/runfiles.py

```diff
@@ -339,6 +342,10 @@ def Create(env: Optional[Dict[str, str]] = None) -> Optional["Runfiles"]:
         directory = env_map.get("RUNFILES_DIR")
         if directory:
             return CreateDirectoryBased(directory)
+        
+        directory =  _FindPythonRunfilesRoot()
```

Comment:
This may not find the correct runfiles tree: It's an internal helper meant to find the tree that contains the Python files, solely for the purpose of identifying the calling repo. This tree could be entirely different from the runfiles tree, e.g. on Windows when using a Python ZIP.